src/ansys/mapdl/core/plotting/plotting.py

```diff
@@ -49,31 +48,33 @@
     "FY",
     "FZ",
     "AMPS",
-    "CHRG",
+    "CHRGS",
```

Comment:
This change breaks the plotting ...  I wil check why...

Reply:
This change is not recomended because that how MAPDL returns the info in the listing:

```pycon
>>> print(mapdl.flist())
LIST NODAL FORCES FOR SELECTED NODES         1 TO        3 BY        1
 CURRENTLY SELECTED NODAL LOAD SET= FX   FY   FZ   HEAT CHRG
   *****MAPDL VERIFICATION RUN ONLY*****
     DO NOT USE RESULTS FOR PRODUCTION
 
    NODE  LABEL     REAL           IMAG
        1  CHRG    10.0000000      0.00000000    
        2  CHRG    10.0000000      0.00000000    
        3  CHRG    10.0000000      0.00000000
```

Please @AlejandroFernandezLuces get back ``CHRG`` label.